src/dodal/beamlines/p99.py

```diff
@@ -0,0 +1,62 @@
+from dodal.beamlines.beamline_utils import device_instantiation
+from dodal.beamlines.beamline_utils import set_beamline as set_utils_beamline
+from dodal.devices.p99.sample_stage import FilterMotor, SampleAngleStage
+from dodal.devices.stages import ThreeAxisStage
+from dodal.log import set_beamline as set_log_beamline
+from dodal.utils import get_beamline_name
+
+BL = get_beamline_name("BL99P")
+set_log_beamline(BL)
+set_utils_beamline(BL)
+
+
+def sample_angle_stage(
+    wait_for_connection: bool = True, fake_with_ophyd_sim: bool = False
+) -> SampleAngleStage:
+    """Sample stage for p99"""
+
+    return device_instantiation(
+        SampleAngleStage,
+        prefix="-MO-STAGE-01:",
+        name="sample_angle_stage",
+        wait=wait_for_connection,
+        fake=fake_with_ophyd_sim,
+    )
+
+
+def sample_stage_filer(
+    wait_for_connection: bool = True, fake_with_ophyd_sim: bool = False
+) -> FilterMotor:
+    """Sample stage for p99"""
+
+    return device_instantiation(
+        FilterMotor,
+        prefix="-MO-STAGE-02:MP:SELECT",
+        name="sample_stage_filer",
+        wait=wait_for_connection,
+        fake=fake_with_ophyd_sim,
+    )
+
+
+def sample_xyz_stage(
+    wait_for_connection: bool = True, fake_with_ophyd_sim: bool = False
+) -> ThreeAxisStage:
+    return device_instantiation(
+        FilterMotor,
+        prefix="-MO-STAGE-02:LAB:",
+        name="sample_xyz_stage",
+        wait=wait_for_connection,
+        fake=fake_with_ophyd_sim,
+    )
+
+
+def sample_lab_xyz_stage(
+    wait_for_connection: bool = True, fake_with_ophyd_sim: bool = False
+) -> ThreeAxisStage:
+    return device_instantiation(
+        FilterMotor,
+        prefix="-MO-STAGE-02:LAB:",
```

Comment:
Should: This has the same prefix as `sample_xyz_stage` so presumably is writing to the same PVs. How is it a different device?